earth_enterprise/docker/README.md

```diff
@@ -0,0 +1,167 @@
+# Docker Image for Running GE Open Source Earth Server and Fusion Server
+
+## Setting up Docker
+
+### Ubuntu
+
+Install Docker as root:
+
+```BASH
+# apt-get install docker.io
+```
+
+Add users that you want to have access to Docker to the `docker` group:
+
+```BASH
+# adduser muser docker
+```
+
+
+## Pulling an Image from Docker Cloud
+
+You can get a pre-built image from a Docker registry.
+
+```BASH
+docker pull thermopylae/opengee-experimental
+```
+
+
+## Running the Container
+
+### Running the Open GEE Servers
+
+#### Running Open GEE Server with Persistent Globe and Map Databases
+
+In order to be able to build and serve Earth databases that persist between 
+instantiations of the Docker image, you need a persistent assets directory 
+outside of the Docker container.
+
+To run the Docker container using a persistent globe assets directory outside 
+the container, use the following script.  Set the `HOST_GEVOL_PATH`
+environment variable to the path outside the container where you want to store
+your persistent assets.
+
+```BASH
+HOST_GEVOL_PATH=/persistent/gevol/ ./bin/start-docker-image.sh
+```
+
+You can optionally set the `GEE_SERVER_PORT` and even  `IMAGE_TAG` to override
+the default values.
+
+
+##### Initializing a New Location for Persistsing Databases
+
+If you want to create the directory structure needed for storing globe and map
+databases in a new location, you can use the following commands (after
+substituting the desired path), and follow the prompts.
+
+```BASH
+sudo mkdir -p /persistent/gevol/assets
+docker run --cap-add=DAC_READ_SEARCH -v /persistent/gevol:/gevol -ti opengee-experimental /opt/google/bin/geconfigureassetroot --new --assetroot /gevol/assets
```

Comment:
I had some problems with permissions when I used the existing asset root on my machine. (And, obviously, the master host has to be switched.)  I haven't done enough testing with it, but it seems that I may work.